.pre-commit-config.yaml

```diff
@@ -1,5 +1,12 @@
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
+  repos:
+    # This should be before any formatting hooks like isort
```

Comment:
Are you planning to add isort to the template or will we leave that until we adopt ruff more?

Reply:
I am planning to add isort, I started putting this pre-commit here then doubted whether I should. It definitely belongs in the template, but is it needed here? Probably?

Reply:
I think we need to use isort not ruff as ruff doesn't support length sorting the stdlib and I don't want to generate the churn (especially on core) of changing that.